inspectit-oce-documentation/src/docs/asciidoc/_includes/monitoring-environment/common-tags.adoc

```diff
@@ -0,0 +1,69 @@
+=== Common Tags
+
+inspectIT OCE provides a map of common tag keys and values that are used when recording all metrics.
+This features enables to "label" metrics collected with inspectIT OCE and share common information about a process or an environment where the process runs.
+The map can be <<User Defined Tags Provider,extended or overwritten by the user>> when this is required.
+
+Tags providers are responsible for extracting information from a specific source and provide a map of key value pairs that are then combined into the common tag context.
+Each provider specifies a priority and if same tag keys are provided by two providers, then the tag value from a provider with higher priority will be used.
```

Comment:
.. and if **the** same tag keys are provided by two providers, then the tag value from **the** provider with higher priority will be used.